concourse/pipelines/create-bosh-concourse.yml

```diff
@@ -1042,6 +1021,34 @@ jobs:
               params:
                 file: generated-concourse-ssh-keys/concourse_id_rsa
 
+      # FIXME: remove this when it's run everywhere.
+      - task: extract-acm-cert-arn
+        config:
+          platform: linux
+          image_resource: *awscli-image-resource
+          params:
+            AWS_DEFAULT_REGION: ((aws_region))
+          inputs:
+            - name: concourse-tfstate
+          outputs:
+            - name: system-cert
+          run:
+            path: sh
+            args:
+            - -e
+            - -c
+            - |
+              tf_cert_details=$(jq -r '.modules[].resources."aws_acm_certificate.system"' concourse-tfstate/concourse.tfstate)
+              if [ "${tf_cert_details}" = "null" ]; then
+                echo "Finding cert to import"
+                aws acm list-certificates \
```

Comment:
Creating a new environment failed, because the `aws acm list-certificates` returns an empty string if the certificate doesn't exist. We have to handle this case too.

Reply:
Added a check for non-existing cert.